README.md

```diff
@@ -31,7 +31,11 @@ These are the available plugins in this repository.
 
 ## Issues
 
-Please check existing issues and file any new issues, bugs, or feature requests in the [flutter-plugin issue list](https://github.com/cph-cachet/flutter-plugins/issues).
+- Please check existing issues and file any new issues, bugs, or feature requests in the [flutter-plugin issue list](https://github.com/cph-cachet/flutter-plugins/issues).
+
+- From Android 14 we need to upgrade the version of Health Connect SDK for the [Health Package](https://github.com/hibitness/flutter-plugins/tree/master/packages/health). It is a prerequisite for Android 14 and apps will not be able to integrate with Health Connect on Android 14 without it.
+
+  Refer link: [Health Connect](https://developer.android.com/jetpack/androidx/releases/health-connect#1.1.0-alpha01)
```

Comment:
here
    // TODO: Checking this SDK & SDK in health package for Android 14
    // https://developer.android.com/jetpack/androidx/releases/health-connect#1.1.0-alpha03

Reply:
@hibitness-quan-nguyen2 
Note 1:
If you never need to modify flutter-plugins, you must directly utilize the https://pub.dev/packages/health package in your project without involving "flutter-plugins".
See this ticket why modified flutter-plugins 
1. https://ntvict.atlassian.net/browse/DEV-3323.
2. TODO After any new changes add ticket link here

Note 2:
In case "flutter-plugins" does not promptly support Android 14, you must adopt the following library for Android to ensure compatibility with Android 14:
For more information, please refer to this link: [Health Connect](https://developer.android.com/jetpack/androidx/releases/health-connect#1.1.0-alpha01).

Reply:
replace above text in 34 to 38 line